app/models/forms/bulk_upload_sales/guidance.rb

```diff
@@ -16,7 +16,7 @@ def back_path
       end
 
       def old_template_path
```

Comment:
might be worth calling this "legacy_template_path" if that's the term we use in the service

Reply:
( I think it may have been me that called it old originally )

Reply:
Sure! Will fix, thanks!